test/framework/k8s_utils.go

```diff
@@ -523,3 +526,81 @@ func DeployRootPod(ctx context.Context, c client.Client, namespace string, noden
 	}
 	return &rootPod, nil
 }
+
+// DeployZeroDownTimeValidatorJob deploys k8s job in cluster, which ensures
+// zero down time by continuously checking kube-apiserver health.
+// This job fails once health check fails and associated pod results in error status.
+func DeployZeroDownTimeValidatorJob(ctx context.Context, c client.Client, testName, namespace, token string) (*batchv1.Job, error) {
```

Comment:
Please do not extend the `framework` package if not needed. Move this function to the `highavailability` package